File: Python/Warmup-1/mix_start.py

# this function return true if the given string begins with "mix", except the 'm' can be anything
def mix_start(str):
	
	b = False

	if len(str) >= 3 and str[1:3] == "ix":
		b = True

	return b

print(mix_start("mix snacks"))
print(mix_start("pix snacks"))
print(mix_start("piz snacks"))

File: Java/Warmup-1/mixStart.java

public class warmup1 {

	// this function return true if the given string begins with "mix", except the 'm' can be anything
	public boolean mixStart(String str) {
		
		boolean bool = false;
		
		if (str.length() >= 3 && str.charAt(1) == 'i' && str.charAt(2) == 'x')
			bool = true;
		
		return bool;
		
	}
	
	public static void main(String[] args) {
		
		warmup1 o = new warmup1();
		
		System.out.println(o.mixStart("mix snacks"));
		System.out.println(o.mixStart("pix snacks"));
		System.out.println(o.mixStart("piz snacks"));
		
	}
	
}
